src/Sankey.js

```diff
@@ -0,0 +1,201 @@
+import {sankey, sankeyLinkHorizontal} from "d3-sankey";
+
+import {accessor, assign, configPrep, constant, elem} from "d3plus-common";
+import {Path, Rect} from "d3plus-shape";
+import {dataLoad as load, Viz} from "d3plus-viz";
+
+/**
+    @class Rings
+    @extends external:Viz
+    @desc Creates a sankey visualization based on a defined set of XXXXX. [Click here](xxxxxx) for help getting started using the Sankey class.
```

Comment:
create a Sankey example and link to it here! take a look at the other example and let me know if you have questions (don't worry about publishing the example, I'll take care of that when we release)